python/semantic_kernel/connectors/memory/chroma/chroma_memory_store.py

```diff
@@ -46,7 +46,7 @@ def __init__(
             # Create a ChromaMemoryStore with a custom Settings instance
             chroma_remote_data_store = ChromaMemoryStore(
                 client_settings=Settings(
-                    chroma_api_impl="rest",
+                    chroma_api_impl="chromadb.api.fastapi.FastAPI"",
```

Comment:
Hi @GaiusRed, you mentioned in https://github.com/microsoft/semantic-kernel/issues/3046 that there is a "workaround" by configuring `chroma_api_impl` to be `chromadb.api.fastapi.FastAPI` instead of `rest`. Is that the correct way to configure it? I am seeing [ChromaDB's docs](https://docs.trychroma.com/migration ) specify that `chroma_api_impl` should be `rest`.

Reply:
Hello @moonbox3! From the docs you linked:
```
### http client (to talk to server backend)

# before
import chromadb
from chromadb.config import Settings
client = chromadb.Client(Settings(chroma_api_impl="rest",
                                        chroma_server_host="localhost",
                                        chroma_server_http_port="8000"
                                    ))

# after
import chromadb
client = chromadb.HttpClient(host="localhost", port="8000")
```
If you are already using "rest" implementation, the correct way to migrate is to stop using the Client base object and use the HttpClient object instead.

If you check how the HttpClient object is implemented, it just configures the Client's Settings to use the `chromadb.api.fastapi.FastAPI` instead of `rest`:
```
def HttpClient(
    host: str = "localhost",
    port: str = "8000",
    ssl: bool = False,
    headers: Dict[str, str] = {},
    settings: Settings = Settings(),
) -> API:
    settings.chroma_api_impl = "chromadb.api.fastapi.FastAPI"
    settings.chroma_server_host = host
    settings.chroma_server_http_port = port
    settings.chroma_server_ssl_enabled = ssl
    settings.chroma_server_headers = headers
    return Client(settings)
```
Note that I just followed how #2050 solved the persistent client issue (use the correct Settings on the base Client object instead of using the PersistentClient as per the migration document).